packages/react/src/components/Dialog/Dialog.stories.js

```diff
@@ -171,3 +211,238 @@ NonModal.args = {
+            magna. Proin ornare tellus quis hendrerit egestas. Donec pharetra
+            leo nec molestie sollicitudin.
+          </p>
+          <br></br>
+          <TextInput
+            id="dialog-text-input"
+            labelText="Name"
+            placeholder="Enter your name"
+            style={{ marginBottom: '1rem' }}
+          />
+          <Select
+            id="dialog-select"
+            labelText="Region"
+            defaultValue="us-south"
+            style={{ marginBottom: '1rem' }}>
+            <SelectItem value="us-south" text="US South" />
+            <SelectItem value="us-east" text="US East" />
+          </Select>
+          <p>
+            Elit hic at labore culpa itaque fugiat. Consequuntur iure autem
+            autem officiis dolores facilis nulla earum! Neque quia nemo sequi
+            assumenda ratione officia Voluptate beatae eligendi placeat nemo
+            laborum, ratione.
+          </p>
+          <p>
+            Elit hic at labore culpa itaque fugiat. Consequuntur iure autem
+            autem officiis dolores facilis nulla earum! Neque quia nemo sequi
+            assumenda ratione officia Voluptate beatae eligendi placeat nemo
+            laborum, ratione.
+          </p>
+          <br></br>
+          <TextInput
+            id="dialog-text-input"
+            labelText="Name"
+            placeholder="Enter your name"
+            style={{ marginBottom: '1rem' }}
+          />
+          <Select
+            id="dialog-select"
+            labelText="Region"
+            defaultValue="us-south"
+            style={{ marginBottom: '1rem' }}>
+            <SelectItem value="us-south" text="US South" />
+            <SelectItem value="us-east" text="US East" />
+          </Select>
+          <p>
+            Elit hic at labore culpa itaque fugiat. Consequuntur iure autem
+            autem officiis dolores facilis nulla earum! Neque quia nemo sequi
+            assumenda ratione officia Voluptate beatae eligendi placeat nemo
+            laborum, ratione.
+          </p>
+        </Dialog.Body>
+        <Dialog.Footer>
+          <Button
+            type="button"
+            kind="secondary"
+            onClick={closeDialog}
+            autoFocus>
+            Close
+          </Button>
+          <Button type="button" kind="primary" onClick={closeDialog}>
+            Save
+          </Button>
+        </Dialog.Footer>
+      </Dialog>
+    </div>
+  );
+};
+WithScrollingContent.args = {
+  modal: true,
+  open: true,
+};
+
+export const PassiveDialog = ({ open: _open, ...args }) => {
+  const [open, setOpen] = useState(_open);
+
+  function toggleDialog() {
+    setOpen(!open);
+  }
+
+  function closeDialog(e) {
+    setOpen(false);
+  }
+
+  function handleRequestClose(e) {
+    action('Dialog onRequestClose');
+    closeDialog(e);
+  }
+
+  useEffect(() => {
+    setOpen(_open);
+  }, [_open]);
+
+  return (
+    <div>
```

Comment:
These could be fragments instead of wrapper divs
```suggestion
    <>
```